test/boost/conftest.py

```diff
@@ -0,0 +1,261 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import io
+import logging
+import os
+import subprocess
+import sys
+import time
+from pathlib import Path, PosixPath
+from typing import Dict, Sequence, Union, Optional
```

Comment:
You can use `dict` instead of `Dict` , `T1 | T2` instead of `Union` and `T | None` instead of `Optional`
Also `typing.Sequence` is deprecated, please, use `collections.abc.Sequence` instead.